Utils/layouts/head.js

```diff
@@ -14,6 +13,23 @@ const AppHead = ({ path, t }) => {
     pageTitle = t("head.page.landing");
   }
 
+  const $ = "window";
```

Comment:
I'm not sure this is right: I'm pretty sure the `$` is standing in for JQuery. You can replace `window` with any value and the result is the same as far as I can see.